src/fsharp/FSharp.Core.nuget/FSharp.Core.LatestNuget.nuspec

```diff
@@ -3,10 +3,10 @@
     <metadata>
         <!-- rename to FSharp.Core when done -->
         <id>FSharp.Core</id>
-        <title>FSharp.Core for F# 4.1</title>
-        <summary>FSharp.Core for F# 4.1</summary>
+        <title>FSharp.Core for F# 4.3</title>
```

Comment:
This update is badly wrong.  It should be "FSharp.Core for F# 4.1" or "FSharp.Core 4.4.3.0".